test_challenger.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import pytest
+
+from numpy import testing as npt
+from sdp import challenger_sdp
```

Comment:
[As suggested](https://github.com/stumpy-dev/sliding_dot_product/pull/13#discussion_r1976654151), 
will update this by testing modules that have both `sliding_dot_product ` & `setup`, for range 3 to 13 (inclusive).